Geocoder/src/main/java/com/example/Geocoder/CensusService.java

```diff
@@ -0,0 +1,64 @@
+package com.example.Geocoder;
+
+import java.net.URI;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.DefaultUriBuilderFactory;
+
+import reactor.core.publisher.Mono;
+
+/**
+ * The service layer for the Geocoder application. This class is responsible for
+ * making requests to the US Census Geocoder API and returning the response as JSON.
+ * 
+ * <p> Use Case: Validate an American address as well as return longitude and latitude<p>
+ * <p> Return: JSON response from US Census API<p>
+ * @author Jabir Emeka
+ * @version 1.0
+ * @param address
+ */
+
+@Service
+public class CensusService {
+
+    private final WebClient webClient;
+    private final DefaultUriBuilderFactory uriBuilderFactory;
+
+    // https://geocoding.geo.census.gov/geocoder/Geocoding_Services_API.pdf
+    // https://www.baeldung.com/spring-autowire
+
+    @Autowired //injecting the WebClient.Builder
```

Comment:
Question: I don't believe `@Autowired` is required here as the public constructor will automatically pull in available beans.